resilience4j-spring-cloud/src/main/java/io/github/resilience4j/ratelimiter/autoconfigure/RefreshScopedRateLimiterAutoConfiguration.java

```diff
@@ -0,0 +1,17 @@
+package io.github.resilience4j.ratelimiter.autoconfigure;
+
+import io.github.resilience4j.ratelimiter.RateLimiter;
+import org.springframework.boot.autoconfigure.AutoConfigureAfter;
+import org.springframework.boot.autoconfigure.AutoConfigureBefore;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.cloud.autoconfigure.RefreshAutoConfiguration;
+import org.springframework.cloud.context.scope.refresh.RefreshScope;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConditionalOnClass({RateLimiter.class, RefreshScope.class})
+@AutoConfigureAfter(RefreshAutoConfiguration.class)
+@AutoConfigureBefore(RateLimiterAutoConfiguration.class)
+public class RefreshScopedRateLimiterAutoConfiguration extends AbstractRefreshScopedRateLimiterConfiguration {
+
```

Comment:
@dlsrb6342  you do not need to do the same as circuit breaker below for registering health indicator ? as Rate limiter has health indicator registration as well as u can see in RateLimiterAutoConfiguration  for spring boo1 and 2

Reply:
@Romeh Yes good point. It is needed. But I think with #571, HealthIndicator registration can be removed on CircuitBreaker and RateLimiter.

Reply:
@dlsrb6342  till we can finish and merge 571, let us stay consistent here please if possible so add it to rate limiter and when u create PR for 571 . then u can clean them properly with proper test/code/dependency setup in place :)  !

Reply:
Ok I'll make RateLimiter like CircuitBreaker

Reply:
@Romeh  I tried to make refreshable `HealthIndicator` both `CircuitBreaker` and `RateLimiter`. But current HealthIndicators are not beans, so it is impossible to make it refreshable in all cases(spring-web, spring-webflux, spring-boot1, spring-boot2 and so on).  

I think in spring-cloud, we should remove `HealthIndicator`. And refactoring `HealthIndicator` on #571.

Reply:
Since the other PR is merged now, would you like to adapt it now?